jbang/aws-bedrock-ingestion/README.adoc

```diff
@@ -0,0 +1,199 @@
+== AWS Bedrock Knowledge Base example
+
+In this sample you'll use the AWS Bedrock component.
+
+Through the usage of EventBridge and SQS Services you'll be able to consume events from specific bucket and start ingestion job on your knowledge base.
+
+=== Install JBang
+
+First install JBang according to https://www.jbang.dev
+
+When JBang is installed then you should be able to run from a shell:
+
+[source,sh]
+----
+$ jbang --version
+----
+
+This will output the version of JBang.
+
+To run this example you can either install Camel on JBang via:
+
+[source,sh]
+----
+$ jbang app install camel@apache/camel
+----
+
+Which allows to run CamelJBang with `camel` as shown below.
+
+=== Setup the AWS S3 bucket, SQS Queue and EventBridge Rule through Terraform
+
+If you are in a hurry you can also try this example by running the Terraform configuration provided in Terraform folder.
+
+[source,sh]
+----
+cd terraform/
+----
+
+and then run
+
+[source,sh]
+----
+terraform init
+----
+
+At this point you should be to run the configuration
+
+[source,sh]
+----
+terraform apply -var="s3_bucket_name=s3-bedrock-test-123" -var="sqs_queue_name=sqs-bedrock-test-123"
+----
+
+You can specify whatever bucket name or SQS name you want.
+
+At the end the AWS environment on your account will be completed, and you could go ahead with the example.
+
+Now you should be able to create a knowledge base on your AWS Bedrock console by pointing to the just created S3 bucket.
+
+Don't forget to specify the correct sqs ARN in the application.properties and add the correct knowledge base Id and data source Id for AWS Bedrock.
+
+=== How to run
+
+Then you can run this example using:
+
+[source,sh]
+----
+$ jbang -Dcamel.jbang.version=4.6.0-SNAPSHOT camel@apache/camel run --properties=application.properties Counter.java BedrockPrompt.java
```

Comment:
here, the 4.6.0-SNAPSHOT is used and in the Java file, it is mentioning 4.5.0-SNAPSHOT.
Is it possible to align them? or there is a technical reason preventing it?

Reply:
Fixes with this: https://github.com/apache/camel-kamelets-examples/pull/79/files